typescript/infra/config/environments/mainnet2/chains.ts

```diff
@@ -40,12 +40,26 @@ export const mainnetConfigs: ChainMap<ChainMetadata> = {
   gnosis: chainMetadata.gnosis,
 };
 
+// Blessed non-Ethereum chains.
+export const nonEthereumMainnetConfigs: ChainMap<ChainMetadata> = {
```

Comment:
Should this be `sealevelMainnetConfigs` or could other non-evm chains possibly go here? Regardless, I guess a lot of this is temporary anyway